INC CREATEv1

```
@@ -0,0 +1,197 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support.ui import Select
+from datetime import date
+from selenium.webdriver.common.action_chains import ActionChains
+import unittest
+
+class Servicenow_IncCreate(unittest.TestCase):
+    def setUp(self):
+        self.driver = webdriver.Firefox()
+#        base_url =  "https://dev65721.service-now.com"
+        
+#        driver.implicitly_wait(10)
+
+#        
+
+#        driver.get(base_url)
+
+    def test_selenium(self):	
+        #
+        identifier = "AUTOMATED TESTING"
+        module = "INCIDENT"
+        action = "CREATE"
+        job_name = ""
+        today = str(date.today())
+        
+        driver = self.driver
+        
+        base_url =  "https://dev65721.service-now.com"
+        
+        driver.get(base_url)
+        
+        
+        driver.switch_to_frame("gsft_main")
+        username = driver.find_element_by_id("user_name")
+        username.send_keys("admin")
+
+        password = driver.find_element_by_id("user_password")
+        password.send_keys("Liverpool28")
+        password.send_keys(Keys.RETURN)
+
+        identifier_inc = ("AUTOMATED TESTING - INCIDENT - %s" %today)
+
+        # depreciated warning
+        driver.switch_to.default_content()
+        # driver.switch_to.frame
+
+        time.sleep (10)
+
+        # Left Nav, incident.do 
+        createnew = driver.find_element_by_id("filter")
+        createnew.send_keys("Create New")
+        time.sleep(1)
+        createnew.send_keys(Keys.RETURN)
+        time.sleep(1)
+
+        # switch to main frame to create the incident
+        driver.switch_to_frame("gsft_main")
+
+
+        try:
+            element = WebDriverWait(driver, 10).until(
+            EC.presence_of_element_located((By.ID, "incident.category"))
+                
+                )
+        except:
+            print("Element not found")
```

This is swallowing stack, maybe error and dump the stack